e2e-tests/cypress/tests/integration/channels/collapsed_reply_threads/following_spec.ts

```diff
@@ -272,3 +276,12 @@ function scrollThreadsListToEnd(maxScrolls = 1, scrolls = 0) {
     });
 }
 
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            postMessageWithReply: typeof postMessageWithReply;
+            scrollThreadsListToEnd: typeof scrollThreadsListToEnd;
+        }
+    }
+}
```

Comment:
This should be removed since this will become available globally to `cy.*` which will throw an error for they're not valid commands.